.github/workflows/ci-scripts-build.yml

```diff
@@ -51,6 +51,23 @@ jobs:
       matrix:
         # Job names also name artifacts, character limitations apply
         include:
+          - os: ubuntu-22.04
+            cmp: gcc-12
+            name: "Ub-22 gcc-12 c++20"
```

Comment:
It might help maintainers to put `Werror` in the name, to make it less surprising, like PVXS does: https://github.com/mdavidsaver/pvxs/blob/master/.github/workflows/ci-scripts-build.yml#L43

Reply:
> put Werror in the name

Done.